src/control/server/storage/bdev.go

```diff
@@ -331,6 +331,12 @@ func (ncs NvmeControllers) Update(ctrlrs ...*NvmeController) NvmeControllers {
 	return ncs
 }
 
+// NvmeAioDevice returns struct representing an emulated NVMe AIO device (file or kdev).
```

Comment:
Seems to be only used with storage/mocks.go => Should it not be better to put in the mock file ?

Reply:
these are intended to be used in backend in following PRs so I would prefer to keep them here, the idea is to improve the code quality and test coverage for using emulated NVMe and these types will help to facilitate that

Reply:
More than OK, if it could help to improve the code quality and coverage :)